src/main/java/com/_up/megastore/controllers/interfaces/IUserController.java

```diff
@@ -15,4 +13,10 @@ public interface IUserController {
   void activateUser(@PathVariable UUID userId,
       @RequestBody @Valid ActivateUserRequest activateUserRequest);
 
+  @PostMapping("/recover-password/send-email")
+  void sendEmailToRecoverPassword(@RequestParam String email);
+
+  @PostMapping("/{userId}/recover-password")
+  void recoverPassword(@PathVariable UUID userId, @RequestParam String newPassword, @RequestParam UUID recoverPasswordToken);
```

Comment:
Acá podríamos usar un body en vez de recibir como parámetros la password y el token
 ```suggestion
  @PostMapping("/{userId}/recover-password")
  void recoverPassword(@PathVariable UUID userId, @RequestBody RecoverPasswordRequest recoverPasswordRequest);
```